ci/pipeline.yml

```diff
@@ -184,6 +184,8 @@ jobs:
       resource: logsearch-for-cloudfoundry-final-builds-dir-tarball
     - get: releases-dir-tarball
       resource: logsearch-for-cloudfoundry-releases-dir-tarball
+  - task: install-node-modules
+    file: logsearch-config/ci/install-node-modules.yml
```

Comment:
It might be cleaner not to use a file. Then we don't have to redeclare the registry-image in the YML file and we can just specify the command in the pipeline